getting-started/cloudflare-workers.md

```diff
@@ -373,4 +373,36 @@ then edit `wrangler.toml`, and add this code after `compatibility_date` line.
 main = "src/index.ts"
 minify = true
 ```
+
 Everything is ready! Now push the code and enjoy it.
+
+## Load env when local development
+
+To configure the environment variables for local development, create the `.dev.vars` file in the root directory of the project.
+Then configure your environment variables as you would with a normal env file.
+
+```
+SECRET_KEY=value
+API_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9
+```
+
+
+> For more about this section you can find in the Cloudflare documentation:
+> https://developers.cloudflare.com/workers/wrangler/configuration/#secrets
+
+Then we use the [env adapter](/helpers/adapter#env) to get the environment variables in our code.
```

Comment:
We recommend using the `c.env` like a `c.env.SECRET_KEY` instead of the env adapter.

Reply:
@yusukebe updated🙂